dbt/adapters/databricks/connections.py

```diff
@@ -414,6 +427,7 @@ def open(cls, connection: Connection) -> Connection:
                     catalog=creds.database,
                     # schema=creds.schema,  # TODO: Explicitly set once DBR 7.3LTS is EOL.
                     _user_agent_entry=user_agent_entry,
+                    http_headers=http_session_headers,
```

Comment:
Then it will be `TypeError`, what do we want in this case? Merge them with respect to specified by user (if our and their contains the same key, we take user's)?

Reply:
`http_headers` should have type 'Optional[List[Tuple[str, str]]', but seems that user can't pass such type in connection_parameters: I tried to write tests using `config_from_parts_or_dicts`, but got error: `dbt.exceptions.DbtConfigError: Runtime Error
E             in _deep_map_render, expected one of (<class 'list'>, <class 'dict'>, <class 'int'>, <class 'float'>, <class 'str'>, <class 'NoneType'>, <class 'bool'>), got <class 'tuple'>`.
So should we just rely on this and assume that user can't pass http_headers (if passed, then we should raise exception) or allow user to pass Dict[str, str] instead List[Tuple[str, str]]?

Reply:
cc @ueshin

Reply:
Ah, then, we need to handle `http_headers` to accept `dict`. Let me make it in a separate PR first.

Reply:
I submitted a PR #174.

Reply:
I merged the PR. Could you update this PR? Thanks.

Reply:
Updated